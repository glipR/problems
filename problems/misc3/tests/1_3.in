5 5
S....
...B.
.....
.A...
....G
