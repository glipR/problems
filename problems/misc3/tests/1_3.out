165.68181818181807
